plugins/woocommerce/includes/emails/class-wc-email.php

```diff
@@ -407,6 +407,11 @@ public function get_default_additional_content() {
 	 * @return string
 	 */
 	public function get_additional_content() {
+		$additional_content = $this->get_option( 'additional_content' );
+		if ( ! is_string( $additional_content ) ) {
```

Comment:
One thing I'm curious about is when this condition that tests if `$additional_content` is not a string might be satisfied (or if it is needed, or if it makes sense).

Suppose the merchant has never touched the email settings, and they then manually send the order details to the customer: the call on the previous line (to `$this->get_option( 'additional_content' )`) is enough to fetch the default additional content without this block.

Reply:
You're right. I was doing some testing on the defaults `$this->get_option()` uses (not just for the "additional content" setting) but in general and probably got confused. That is, for all other e-mail settings (subject, heading, etc.), the default is an empty string (`''`) but for "additional content" the default is... well... the default additional content. That's a bit confusing as we do want to use the default for the other settings when the field is empty but not for the additional content.

Anyways, I've simplified things in a27815bbee01b96701afbe28f7ced83b3e4a334. Hope that's what you meant.

Reply:
Yes, that's exactly it (in fact, it's even cleaner/more concise than I was thinking). Thank you!